Makefile

```
@@ -218,11 +223,14 @@ docs-preview: ## Preview the documentation through mkdocs
 docs-build:  ## Build the documentation to the `site/` directory
 	mkdocs build --strict
 
+gen.crd-docs: generate-docs-crds
 generate-docs-crds: ## Build the documentation for CRD
 	@GOBIN=$(GOBIN) build/crds/generate-crd-docs.sh
 
+generate: gen.codegen gen.crds gen.rbac gen.docs gen.crd-docs
```

Let's add some doctext to the top level command to help users as well. I'm less concerned about doctext for other `gen.` targets, and they could even make the help text a little too verbose.

```suggestion
generate: gen.codegen gen.crds gen.rbac gen.docs gen.crd-docs ## DOCTEXT
```